auth/lib/models/code.dart

```diff
@@ -216,7 +251,13 @@ class Code {
         secret.hashCode ^
         type.hashCode ^
         counter.hashCode ^
-        rawData.hashCode;
+        rawData.hashCode ^
+        display.hashCode;
+  }
+
+  @override
+  String toString() {
```

Comment:
Not having a `toString` implementation will ensure we don't accidentally log sensitive data